raspi/soundserver/soundserver.py

```diff
@@ -3,12 +3,16 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 import sys
 import os
+import subprocess
+import logging
+import os
```

Comment:
Could you please reorder these `import` lines, to follow PEP8 coding style? The order should be:

1. Standard libs
2. Third-party libs
3. Our project's libs